tests/test_cmd_base.py

```diff
@@ -193,6 +206,32 @@ def test_execute(self, depfile_name):
             '--db-file', depfile_name,
             '--mine', 'min'])
 
+    def test_execute_with_legacy_dict_loader(self, depfile_name):
+        members = {'task_xxx1': lambda: {'actions': []}}
+
+        class LegacyLoader(TaskLoader):
+            def load_tasks(self, cmd, opt_values, pos_args):
+                return super()._load_from(cmd, members, [])
+
+        mycmd = self.MyCmd(task_loader=LegacyLoader())
+        assert 'min' == mycmd.parse_execute([
+            '--db-file', depfile_name,
+            '--mine', 'min',
+        ])
+
```

Comment:
**I** usually use two blank line between methods if the method as blank lines to separate blocks. Otherwise I use just one line. And if methods are separated by 2 blank lines, I use 3 between classes.

Not PEP8.  But I guess this makes code more readable. Although not always strict about it.